clang/include/clang/Basic/Attr.td

```diff
@@ -1226,6 +1226,19 @@ def SYCLIntelNumSimdWorkItems : InheritableAttr {
   let PragmaAttributeSupport = 0;
 }
 
+def SYCLIntelStallEnable : InheritableAttr {
+  let Spellings = [CXX11<"intel","stall_enable">];
+  let LangOpts = [SYCLIsDevice, SYCLIsHost];
```

Comment:
It looks like we are simply ignoring the attribute on host.  Let's just allow it on device.

Reply:
Done

Reply:
Yes we are ignoring it on host, but now host compiler emits warnings when sees this attribute, because it became unknown for host compiler once you removed `SYCLIsHost` lang option from here. I.e. there is no way to write code with this attribute without emitting warnings now, this is not good for users. Can we return `SYCLIsHost` back, please?

Reply:
I see what you are saying @Fznamznon.  I take back my suggestion.   Thanks!

Reply:
I have added back SYCLIsHost. Thanks for the suggestion.